gemrb/core/GUI/Button.cpp

```diff
@@ -360,16 +360,15 @@ bool Button::IsAnimated() const
 
 bool Button::IsOpaque() const
 {
-	if (animation) {
-		// we are dirty every frame anyway
-		// so no need to get fancy
-		// just redraw everything
-		return false;
-	}
-
+	return false;
```

Comment:
This looks misplaced since I started to notice bugs around buttons.

Reply:
yeah, it was there for testing and I forgot to remove it.